blocks/get_message_info.js

```diff
@@ -0,0 +1,290 @@
+                        break;
+                    case "USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_1":
+                        result = "User Nitro Boost - Level 1";
+                        break;
+                    case "USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_2":
+                        result = "User Nitro Boost - Level 2";
+                        break;
+                    case "USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_3":
+                        result = "User Nitro Boost - Level 3";
+                        break;
+                    case "CHANNEL_FOLLOW_ADD":
+                        result = "Channel Follow Add";
+                        break;
+                    case "GUILD_DISCOVERY_DISQUALIFIED":
+                        result = "Server Discovery Disqualified";
+                        break;
+                    case "GUILD_DISCOVERY_REQUALIFIED":
+                        result = "Server Discovery Requalified";
+                        break;
+                    default:
+                        result = "Unknown";
+                        break;
+                }
+                break;
+            case 29:
+                result = message.url;
+                break;
+            case 30:
+                result = await message.fetchWebhook();
+                break;
+            case 31:
+                result = message.webhookID;
+                break;
+            case 32:
+                result = message.activity && message.activity.partyID;
+                break;
+            case 33:
+                if(!message.activity) {
+                    result = message.activity;
+                } else {
+                    switch(parseInt(message.activity.type)) {
+                        case 1:
+                            result = "Join";
+                            break;
+                        case 2:
+                            result = "Spectate";
+                            break;
+                        case 3:
+                            result = "Listen";
+                            break;
+                        case 5:
+                            result = "Join Request";
+                            break;
+                        default:
+                            result = "Unknown";
+                            break;
+                    }
+                }
+                break;
+            case 34:
+                result = message.application;
+                break;
+            case 35:
+                var obj = {
+                    "Crossposted": "CROSSPOSTED",
+                    "Is Crosspost": "IS_CROSSPOST",
+                    "Suppress Embeds": "SUPPRESS_EMBEDS",
+                    "Source Message Deleted": "SOURCE_MESSAGE_DELETED",
+                    "Urgent": "URGENT",
+                }
+                const flags = message.flags;
+                result = Object.keys(obj).filter(a => flags.has(obj[a]));
+                break;
+            case 36:
```

Comment:
Im just asking why ? 
What about get Message Arguments Block ?
it would be more usefull there